subset/security/readme.md

```diff
@@ -1,48 +1,54 @@
 # Security testing
 
 ## test_tls
-The TLS test attempts to verify various versions of TLS support.  Separate connections will be attempted with different SSL context with the associated TLS support, 1.0, 1.2 and 1.3.
+The TLS test attempts to verify various versions of TLS support.  Separate connections will be attempted with different SSL context with the associated TLS support, 1.0, 1.2 and 1.3 to teset devices server TLS capabilities.  A 5 min max scan period is also used to capture TLS client level capabilities for TLS 1.2 and 1.3.
  
-### Testing procedure
+### Testing procedure for server testing
 After establishing connections to devices, the test will proceed to validate the available certificates with the following criteria:
  1. The certificate is in the x509 format
- 2. The public key length is at least 2048.  Currently handles both RSA and DSA public key formats.
- 3. The certificate is not expired and active for the current date.
-The cipher suite used is also checked but does not currently affect the outcome of the results.  Currently the expected cipher suites are ECDH and ECSA.  If these are not present, a warning message will be logged in the activate.log of the switch node.
+ 2. The public key length is at the correct bit length
+    - If public key is RSA format - Key length must be at least 2048
+    - If public key is EC format - key length must be at least 224
+ 3. The certificate is not expired and active for the current date
+ 4. The cipher suite is valid:(only for TLS < 1.3)
+    - If RSA public key format - No cipher check is required
+    - If EC pubic key format - Check if cipher suites ECDH and ECSA are present
 
- 
-### Note for test developers 
-The functional test code is included in the `tlstest/src/main/java` folder.
+### Testing procedure for client testing
```

Comment:
Read me should give guidance for configuration (e.g. directory to the certificate file and what that certificate file is)

Reply:
Updated, let me know if you think it needs more clarity.